manifests/generated/rbac-operator.authorization.k8s.yaml.in

```diff
@@ -401,6 +401,21 @@ rules:
   - update
   - delete
   - patch
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumes
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - create
+  - delete
```

Comment:
do we need "delete" privileges?  or can the secret get automatically cleaned up for us as a result of ownership references pointing to the vmi?

Reply:
I could do that, but that could leave a lot of secrets laying around in the case of the VMI existing a long time, and many volumes being hotplugged and removed. My initial thought was making the attachmentPod the owner of the secret as that is the scope in which we care about it, but I think that left me with a chicken and egg situation where I can't create the secret because I can't retroactively add an owner, and I can't create the attachmentPod because I don't have the secret yet. I can do some more research here, since I only need the name of the pod, I can create it, it will remain pending until the secret is created, then I can create the secret with an owner. But that will complicate the flow quite a bit.